### PR TITLE
Create/Edit Event

### LINKED ISSUE #43: Create an event [Frontend]
We have a the designs ready for create an event, so we have to change the placeholder UI for creating an event that we have now to the actual designs. For this ticket, we need to modify the **UI modal** to that of the design as below and ensure the changes work with making a create event api call with the new design. 
image

[Figma Link for design](https://www.figma.com/file/LEZGE9yXVag3LY8zHACAY5/Helping-Mamas-%2F-Spr2022?node-id=218%3A15927)

Data changes from the current UI: 
* No shift.
* Max number of volunteers field.
* Ignore "Standard" and "Organization" tabs in the design (we do not have backend for Organization type yet). Keep just one standard for now.

Note: Ignore the drop-down "Distribution" that is present in the design. Move the date fields on its place. 
Note 2: In the button it  should be "Create Event" instead of "Create Shift".
Note 3: If putting in  "Bold", "Italics" etc in the description part of the UI is difficult, it can be skipped. It is not mandatory for now.

Acceptance Criteria:
* UI reflects the designs
* Creating an event with the new frontend works.

### LINKED ISSUE #44: Edit  an event [Frontend]
Follow up of #43. 

Replace the current edit event modal with the the same modal that is being added in #43. The fields will be pre-filled with the details from the event. (Any changes submitted will update the event just like how it currently works now).

Acceptance Criteria:
* Same modal created in #43 is launched to edit the event with pre-filled values.
* Editing an event with the new frontend works.

### PR DESCRIPTION
**Pull request by bphoebew**
UI changes to create and edit events

### CODE REVIEW

File: src/screens/Events/Admin/EventTableRow.jsx [hunk 1]
```diff
@@ -49,25 +49,13 @@ const EventTableRow = ({ event, onEditClicked, onDeleteClicked, idx }) => {
       </td>
       <td>{event.name}</td>
```
Comment (abhigyank) [inline src/screens/Events/Admin/EventTableRow.jsx]: `event.title`

File: src/screens/Events/Admin/EventFormModal.jsx [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+        return event;
+    }
+
+    const emptyStringField = "";
+    const submitText = containsExisitingEvent(event) ? "Submit" : "Create Event";
+    
+    return (
+        <Formik
+        initialValues={{
+            title: containsExisitingEvent(event) ? event.name : emptyStringField,
```
Comment (abhigyank) [inline src/screens/Events/Admin/EventFormModal.jsx]: `event.title`

File: src/screens/Events/Admin/EventFormModal.jsx [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+            date: containsExisitingEvent(event) ? event.date.split("T")[0] : emptyStringField, // strips timestamp
+            startTime: containsExisitingEvent(event) ? event.time : emptyStringField,
```
Comment (abhigyank) [inline src/screens/Events/Admin/EventFormModal.jsx]: `event.startTime`

File: src/screens/Events/Admin/EventFormModal.jsx [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+            endTime: containsExisitingEvent(event) ? event.time : emptyStringField,
```
Comment (abhigyank) [inline src/screens/Events/Admin/EventFormModal.jsx]: `event.endTime`

**Review by abhigyank — CHANGES_REQUESTED** [review]
Really good work! Can you make the small changes? Then its mergable.

**Comment by bphoebew** [comment]
> Really good work! Can you make the small changes? Then its mergable.

Ok I think I made all the small changes!

**Comment by abhigyank** [comment]
Merged! Amazing work @bphoebew  @jasminelli :rocket:

**Pull request merged**